lib/StillImage.tsx

```diff
@@ -51,20 +51,28 @@ export const StillImage: React.FC<StillImageProps> = ({
   const [loaded, unsetLoaded] = useEventState(imgRef, 'load')
 
   useEffect(() => {
-    if (imgRef && imgRef.current) {
-      if (play && src) {
+    if (imgRef.current !== null) {
+      if (play && src !== undefined) {
         imgRef.current.src = src
       } else {
         imgRef.current.src = ''
         unsetLoaded()
       }
     }
-  }, [play, src])
+  }, [play, src, unsetLoaded])
+
+  // keep a stable reference to the external onPlaying callback
+  const __onPlayingRef = useRef(onPlaying)
+  __onPlayingRef.current = onPlaying
```

Comment:
This change seems a bit weird... why? Just ignore the lint rule for dep array if that is the problem.

Reply:
I want to use the latest updated `onPlaying` callback, but not re-run the effect, so I'm keeping a reference to it to be used inside the effect. If it's ignored from the dep array, the callback itself will not be updated.